run-tests.sh

```diff
@@ -24,7 +24,7 @@
 
 
 pep257 invenio_admin && \
-isort -rc -c -df -sp . **/*.py && \
+# isort -rc -c -df -sp . **/*.py && \
```

Comment:
Is isort buggy again ?
Also I noticed a leftover of mine when I was debugging the isort previously, originally it was without ``-sp .`` argument:
``isort -rc -c -df **/*.py``